frontend-ssr/components/SkeletonImage/SkeletonImage.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './SkeletonImage.module.scss';
+
+interface Props {
+  width: string,
+  height: string,
+}
+
+const SkeletonImage = ({ width, height } : Props) => (
+  <li className={styles.skeleton}>
```

Comment:
We should not render a `<li>` here as this forces us to always have an `ul` or `ol` as a parent.  Just render the `div` and depending on the use case we will wrap it into a `<li>` from the outside.